### ISSUE TITLE
When using cp .env.dist .env, nothing happens

### ISSUE THREAD

**Issue by nolanplatt** [mikealmond/hq-trivia-assistant#56]
Hi!

I have followed all steps, just when I use $ cp .env.dist .env, no response happens and no new file is put in folder

Any help is appreciated.
Thanks!

**Comment by Southsideswag16** [comment]
Can you list out everything you have done, in order?

**Comment by nolanplatt** [comment]
Hi, I have done everything - I cloned the repo, cded the repo, installed composer, and did cp .env.dist .env

**Comment by nolanplatt** [comment]
Anyone?

**Comment by tombets2424** [comment]
**When typing...**
```
git clone https://github.com/mikealmond/hq-trivia-assistant .
 cd hq-trivia-assistant
 composer install
 cp .env.dist .env
```
1. Where are you typing it? In gitBash, right? 
2. Are you opening gitBash in the hq-trivia-assistant folder?
3. After you type this does any sort of error happen?
4. Do you have composer and php installed and up to date?
5. Do you have the cacert.pem file installed in either xwamp or wamp64?

If you could answer any of these that could help.

**Comment by Sonishkan** [comment]
what does line 3 do coz when I type that in terminal it just says "-bash: composer: command not found"

**Comment by tombets2424** [comment]
Do you have composer installed?? If not you should install it

**Comment by Sonishkan** [comment]
i Think i have
<img width="572" alt="screen shot 2018-02-12 at 15 52 40" src="https://user-images.githubusercontent.com/36386886/36105566-d8844b56-100c-11e8-86af-e5835c5da089.png">

**Comment by Sonishkan** [comment]
@tombets2424 was that right

**Comment by tombets2424** [comment]
Yeah I'm pretty sure. Do cp .env.dist .env to create the .env file. Then enter your ID and BEARER TOKEN.

**Comment by Sonishkan** [comment]
this is all the files that i have 
<img width="731" alt="screen shot 2018-02-12 at 16 27 08" src="https://user-images.githubusercontent.com/36386886/36107329-976a6a2e-1011-11e8-8821-ad9bf18f8ac1.png">

**Comment by tombets2424** [comment]
You should have .env.dist

**Comment by tombets2424** [comment]
Also are you using gitBash?

**Comment by Sonishkan** [comment]
? im using macos terminal

**Comment by Sonishkan** [comment]
if thats what you mean

**Comment by tombets2424** [comment]
Not sure how this works on Mac? But you should have the file: .env.dist

**Comment by Sonishkan** [comment]
i type the code in but there isnt a file
<img width="578" alt="screen shot 2018-02-12 at 16 47 42" src="https://user-images.githubusercontent.com/36386886/36108332-831a4956-1014-11e8-964c-bf43984616fa.png">

**Comment by tombets2424** [comment]
What this? " cp .env.dist .env "
Download the .env.dist file then type that. It allows you to edit it (the ID and BEARER TOKEN) then save that file as the .env file.

**Comment by Sonishkan** [comment]
where is the file

**Comment by tombets2424** [comment]
You have it when you download the code. If you go to the code page you should see it there. Download everything as a zip.

**Comment by Sonishkan** [comment]
<img width="616" alt="screen shot 2018-02-12 at 16 54 31" src="https://user-images.githubusercontent.com/36386886/36108738-7070c806-1015-11e8-9dae-09f255d573f7.png">

**Comment by Sonishkan** [comment]
when i download it, it disappears

**Comment by tombets2424** [comment]
Hmmmm.... I think I saw about the file disappearing in a different thread but, I can't remember what one.

**Comment by Sonishkan** [comment]
ok ill look on the other ones

**Comment by tombets2424** [comment]
Go to #53 they were on Mac OS and also couldn't locate the env.dist file.

**Comment by Southsideswag16** [comment]
Open the project in a text editor, the .env should be there

**Comment by tombets2424** [comment]
Yeah that's what they did

**Comment by Sonishkan** [comment]
Yh I dont know what to do with the bearings and the game id stuff

**Comment by tombets2424** [comment]
Sorry I've been busy. Do you have your game ID and the BEARER TOKEN?

**Comment by Sonishkan** [comment]
No it’s not working

**Comment by tombets2424** [comment]
What isn't? Getting you id and bearer?

**Comment by Sonishkan** [comment]
The actual run.php file works but i cant get the id and bearer

**Comment by tombets2424** [comment]
Are you using Charles to get the id and bearer. Also if your on android 7.0+ or on some latest iPhone updates you won't be able to do this at all!! Unless you downgrade.

**Comment by Sonishkan** [comment]
im on the latest iOS

**Comment by Sonishkan** [comment]
iOS 11.2.5

**Comment by tombets2424** [comment]
Hmmmm... I don't have an iPhone but I'm pretty sure the latest versions on the iPhone and android can't do this as they have trouble downloading the certificate on their phone from the Charles website.

**Comment by Sonishkan** [comment]
ok

**Comment by Sonishkan** [comment]
is there any other way that i can get the 2 informations

**Comment by Sonishkan** [comment]
i cant connect to the proxy

**Comment by tombets2424** [comment]
No you need to use Charles or some other equivalent application like Charles. You can try something else if you know a similar application but, other then that no, sorry.

**Comment by Sonishkan** [comment]
Basically i when i use the proxy information safari says that its lost network connection

**Comment by tombets2424** [comment]
Yeah that's the problem if you have latest iPhone or Android 7.0 or higher.

**Comment by tombets2424** [comment]
Is your unknown sources enabled?

**Comment by Sonishkan** [comment]
Ive got the certificate on my phone now with the proxy what do i do next

**Comment by Sonishkan** [comment]
Where do i find the id and bearer

**Comment by tombets2424** [comment]
On Charles start a clean (new) session then on your phone open HQ trivia. Click the tabs that appear until you see one that says me. That contains a really long string of letters/numbers for your bearer token and in the actual code there will be an 8 number ID.

**Comment by Sonishkan** [comment]
<img width="904" alt="screen shot 2018-02-12 at 23 04 50" src="https://user-images.githubusercontent.com/36386886/36124751-2ca36d98-1049-11e8-8376-e41da7a1422d.png">
dont know where it says me

**Comment by tombets2424** [comment]
Press the arrow on the second one (api-quiz-hype-space)

**Comment by Sonishkan** [comment]
<img width="904" alt="screen shot 2018-02-12 at 23 07 48" src="https://user-images.githubusercontent.com/36386886/36124835-93d612f4-1049-11e8-9d3b-27905647a795.png">

**Comment by tombets2424** [comment]
Also right click it and enable ssl coding or certificate thing.

**Comment by Sonishkan** [comment]
<img width="907" alt="screen shot 2018-02-12 at 23 09 07" src="https://user-images.githubusercontent.com/36386886/36124882-c12612e0-1049-11e8-8b86-58c64549d787.png">

**Comment by tombets2424** [comment]
For to contents to view the code

**Comment by Sonishkan** [comment]
<img width="873" alt="screen shot 2018-02-12 at 23 11 06" src="https://user-images.githubusercontent.com/36386886/36124961-05e87a94-104a-11e8-82f4-8cef2226df26.png">

**Comment by Sonishkan** [comment]
is that what you mean

**Comment by tombets2424** [comment]
Yes. Have you right clicked on unknown and enabled ssl code or whatever it called?

**Comment by Sonishkan** [comment]
yh

**Comment by Sonishkan** [comment]
<img width="557" alt="screen shot 2018-02-12 at 23 13 09" src="https://user-images.githubusercontent.com/36386886/36125030-4f531d10-104a-11e8-880e-2bbd5c962ac5.png">

**Comment by tombets2424** [comment]
Hold on I'm gonna resources this to see what I did, maybe do raw

**Comment by Sonishkan** [comment]
for which one the first box, second box or both

**Comment by tombets2424** [comment]
Both

**Comment by Sonishkan** [comment]
<img width="900" alt="screen shot 2018-02-12 at 23 15 29" src="https://user-images.githubusercontent.com/36386886/36125113-a330f164-104a-11e8-98ad-dbe345f17d88.png">

**Comment by tombets2424** [comment]
![image](https://user-images.githubusercontent.com/36307300/36125220-07c576a4-104b-11e8-921f-165046cf06dd.png)

this is what I get.

**Comment by tombets2424** [comment]
![image](https://user-images.githubusercontent.com/36307300/36125270-31782604-104b-11e8-93a2-5f75c7842b0e.png)

Then I get this. you can see part of my BEARER TOKEN (at the top) and my ID (In the code)

**Comment by Sonishkan** [comment]
<img width="846" alt="screen shot 2018-02-12 at 23 20 08" src="https://user-images.githubusercontent.com/36386886/36125299-50d8452e-104b-11e8-862b-213a1a634915.png">

**Comment by tombets2424** [comment]
Do users and click me.

**Comment by tombets2424** [comment]
YES!! now copy and paste the BEARER and ID

**Comment by tombets2424** [comment]
do you know how to open/create the .env file?

**Comment by Sonishkan** [comment]
do i copy the word Bearer

**Comment by tombets2424** [comment]
no the long string after it

**Comment by tombets2424** [comment]
double click it then highlight the important part

**Comment by Sonishkan** [comment]
yep

**Comment by tombets2424** [comment]
also delete that picture so other people don't see your ID.

**Comment by Sonishkan** [comment]
ok just trying to find the code to get the file

**Comment by Sonishkan** [comment]
ive done the editing stuff

**Comment by tombets2424** [comment]
so you made the env file and put the BEARER and ID in there? and the env is in the hq-trivia-master folder?

**Comment by tombets2424** [comment]
if so you should be good to run gitBash in the hq-trivia folder then type php run.php and it should say when the next game is.

**Comment by Sonishkan** [comment]
its there but it isnt. if i use an app called atom i can c it but without it i cant

**Comment by tombets2424** [comment]
that shouldn't matter, I don't think. try the code in GitBash

**Comment by Sonishkan** [comment]
what is that for

**Comment by tombets2424** [comment]
what gitBash? that's to run the actual program

**Comment by tombets2424** [comment]
or on MacOS do you use just the terminal?

**Comment by Sonishkan** [comment]
ok

**Comment by Sonishkan** [comment]
ill use the terminal

**Comment by tombets2424** [comment]
I don't know if you can...?

**Comment by Sonishkan** [comment]
<img width="580" alt="screen shot 2018-02-12 at 23 36 03" src="https://user-images.githubusercontent.com/36386886/36125816-8ad3e7b8-104d-11e8-8c32-3ca412843f33.png">

it does but the data it still eastern time

**Comment by Sonishkan** [comment]
<img width="749" alt="screen shot 2018-02-12 at 23 38 12" src="https://user-images.githubusercontent.com/36386886/36125874-d390d3d0-104d-11e8-9db9-79ecd09dd4cc.png">

**Comment by Sonishkan** [comment]
what does this line do $ cp .env.dist .env

**Comment by tombets2424** [comment]
I know its est for everyone. It works. run the code when the lady appears. it might say connected... reconnecting... but don't worry, when the question appears it will show you a possible answer.

**Comment by tombets2424** [comment]
that line was to make the .env file

**Comment by Sonishkan** [comment]
How do i make it work for the UK

**Comment by Sonishkan** [comment]
or is that what it is already but just displayed in est

**Comment by tombets2424** [comment]
it will, I live in the uk. it says est as that's were HQ is based, in the us.

**Comment by tombets2424** [comment]
yes

**Comment by Sonishkan** [comment]
ok

**Comment by Sonishkan** [comment]
how does this actually work, when the game is online

**Comment by tombets2424** [comment]
what you do is go on HQ trivia on your phone. Wait for the lady to appear, then you can do php run.php in the terminal. Instead of it saying when the next broadcast is it will tell you the possible answer. it will display the question and answer in a %. HOWEVER, it can be wrong quite a lot but it does work.

**Comment by Sonishkan** [comment]
ok cheers. I was trying to get a different one working but i messed up. https://github.com/zweg25/HQ

**Comment by Sonishkan** [comment]
https://github.com/zweg25/HQ   this one seems to be better i got it working but then it messed up

**Comment by tombets2424** [comment]
yes I have seen this one a lot more and yes it does seem a lot more reliable. I think I might get the other one and test it to see which is better.

**Comment by Sonishkan** [comment]
when u get it working please tell me 

email me on kingkztuts@gmail.com   please

**Comment by tombets2424** [comment]
sure.

**Comment by Sonishkan** [comment]
cheers mate
